packages/web-components/src/components/content-block/content-block.scss

```diff
@@ -6,7 +6,7 @@
 //
 
 @import 'carbon-components/scss/globals/scss/vars';
-@import '../../globals/scss/vars';
+@import '@carbon/ibmdotcom-styles/scss/globals/vars';
 @import '../../globals/scss/box-sizing';
```

Comment:
@asudoh just checking based on our slack conversation, do we need to reference `@carbon/ibmdotcom-styles` for `box-sizing` too?

Reply:
@jeffchew Good question - Not sure the issue happens with non-Web-Components codebase. The issues was that some elements in Web Components got `box-sizing: content-box` (default in UA style) for some of its element. If we see it happens in non-Web-Components codebase in any way, we should make the move.